tests/test_mds.py

```diff
@@ -5,11 +5,15 @@
 
 import numpy as np
 from numpy.testing import assert_almost_equal
+from sklearn.utils.estimator_checks import check_estimator
 
 from graspologic.embed.mds import ClassicalMDS
 
 
 class TestMDS(unittest.TestCase):
+    def test_sklearn_conventions(self):
```

Comment:
This was kinda hiding in the `test_sklearn.py` file.  I moved it into here and deleted the sklearn one.

Reply:
note: I also purposefully broke ClassicalMDS and ran it again to verify it failed (fit() -> ______fit() = explosions)